migrations/2023-11-22-194806_low_rank_defaults/up.sql

```diff
@@ -0,0 +1,16 @@
+-- Change the hot_ranks to a miniscule number, so that new / fetched content
+-- won't crowd out existing content.
+--
+-- They must be non-zero, in order for them to be picked up by the hot_ranks updater.
+-- See https://github.com/LemmyNet/lemmy/issues/4178
+ALTER TABLE community_aggregates
+    ALTER COLUMN hot_rank SET DEFAULT 0.0001;
```

Comment:
Afaik there was no problem with the default hot rank value.

Reply:
It was less of an issue, but definitely still an issue. This fixes both.